components/jupyter-web-app/kubeflow_jupyter/default/app.py

```diff
@@ -162,7 +162,5 @@ def home():
     nmsps = [base_ns]
     logger.warning("Can't  list namespaces: %s" % api.parse_error(e))
 
-  # todo(apverma): To Be removed after multi-user isolation support is implemented
-  nmsps = [base_ns] # Issue #2937
   return render_template(
-      'notebooks.html', prefix=prefix(), title='Notebooks', namespaces=nmsps)
+    'notebooks.html', prefix=prefix(), title='Notebooks', namespaces=nmsps)
```

Comment:
Could we also get a change that defaults `kubeflow` by default? It was being added here: #3074

Reply:
Sure!